test/unified_format.py

```diff
@@ -1159,6 +1160,24 @@ def _collectionOperation_listIndexes(self, target, *args, **kwargs):
     def _collectionOperation_listIndexNames(self, target, *args, **kwargs):
         self.skipTest("PyMongo does not support list_index_names")
 
+    def _collectionOperation_createSearchIndex(self, target, *args, **kwargs):
+        return target.create_search_index(kwargs["model"])
+
+    def _collectionOperation_createSearchIndexes(self, target, *args, **kwargs):
+        models = [SearchIndexModel(**i) for i in kwargs["models"]]
+        return target.create_search_indexes(models)
+
+    def _collectionOperation_dropSearchIndex(self, target, *args, **kwargs):
+        return target.drop_search_index(*args, **kwargs)
```

Comment:
We can remove this method since it does nothing with the input/output and it can already be found on the collection object. Same for _collectionOperation_createSearchIndex and _collectionOperation_updateSearchIndex I believe.

Reply:
Done